src/test/java/org/vaadin/example/MainViewIT.java

```diff
@@ -28,6 +28,7 @@ private static String getDeploymentHostname() {
     @BeforeEach
     public void open() {
         getDriver().get("http://"+getDeploymentHostname()+":8080/");
+        getCommandExecutor().waitForVaadin();
```

Comment:
Why is this suddenly needed? Is TestBench broken?

Reply:
locally, i can run the test without it.. . but not with TC. 

so i added that one for testing.

Reply:
I think there is a problem that a bundle is built and that returns the building bundle view which testBench waitForVaadin does wait for, but only using `getDriver().get("");` doesn't do waitForVaadin.

```
[15:26:37 ] 2024-02-07T13:26:07.252Z  INFO 20753 --- [  restartedMain] c.v.f.s.frontend.BundleValidationUtil    : Checking if a development mode bundle build is needed
[15:26:37 ] 2024-02-07T13:26:07.278Z  INFO 20753 --- [  restartedMain] c.v.f.s.frontend.BundleValidationUtil    : Dependency @vaadin/hilla-react-crud is missing from the bundle
[15:26:37 ] 2024-02-07T13:26:07.278Z  INFO 20753 --- [  restartedMain] c.v.f.s.frontend.BundleValidationUtil    : Dependency @vaadin/hilla-react-form is missing from the bundle
[15:26:37 ] 2024-02-07T13:26:07.278Z  INFO 20753 --- [  restartedMain] c.v.f.s.frontend.BundleValidationUtil    : Dependency @vaadin/hilla-lit-form is missing from the bundle
[15:26:37 ] 2024-02-07T13:26:07.278Z  INFO 20753 --- [  restartedMain] c.v.f.s.frontend.BundleValidationUtil    : Dependency @vaadin/hilla-react-auth is missing from the bundle
[15:26:37 ] 2024-02-07T13:26:07.279Z  INFO 20753 --- [  restartedMain] c.v.f.s.frontend.BundleValidationUtil    : A development mode bundle build is needed
[15:26:38 ] 2024-02-07T13:26:07.477Z  INFO 20753 --- [onPool-worker-2] c.v.f.s.frontend.TaskUpdatePackages      : Running `npm install` to resolve and optionally download frontend dependencies. This may take a moment, please stand by...
[15:26:38 ] 2024-02-07T13:26:07.872Z  INFO 20753 --- [  restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
```

Reply:
Or then it is this https://github.com/vaadin/hilla/issues/842

Reply:
From the logs it's both as i see it starting a new bundle and restarting the server in the middle of loading.

Reply:
The correct fix would probably be to exclude hilla from the spring-boot-starter dependency.

Reply:
I reverted the waitForVaadin and excluded the hilla dependency (hilla-react) from `vaadin-spring-boot-starter`

Reply:
Sounds like that does not solve any problem but just hides the problem for this repository?

Reply:
Yes. The main problem is that Hilla coming in to Flow spring projects creates issues on maven startup (it runs some strange generators) and the default bundle is missing imports.